### PR TITLE
remove default tipping from UI

### LINKED ISSUE #3137: Remove ability for Creators to set default tipping amounts in their tipping banner
## Description

Currently, Creators can set the set of default tipping amounts in their tipping banner via the Creators dashboard. When present, this will overwrite the general suggested default tipping amounts set by Brave.

image

Browser/client side: https://github.com/brave/brave-browser/issues/24548 

## Solution

- [ ] Step 1: Remove option to set the set of default tipping amounts on the Creator side in the Creators app UI

### PR DESCRIPTION
**Pull request by jlbyrne**
Resolves #3137

### CODE REVIEW

**Review by OAuthBringer — APPROVED** [review]
Simple enough.

File: db/migrate/20230109224336_remove_donations_from_site_banners.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class RemoveDonationsFromSiteBanners < ActiveRecord::Migration[6.1]
+  def up
+    remove_column :site_banners, :default_donation
```
Comment (tsmartt) [inline db/migrate/20230109224336_remove_donations_from_site_banners.rb:3]: I wonder if this will cause any protobuf type errors in the pCDN?

Comment (jlbyrne) [reply db/migrate/20230109224336_remove_donations_from_site_banners.rb:3]: I did edit the protobuf files to remove references to the `default_donations`.  I can run the pCDN job in staging before we push to master to make sure it won't cause more problems though.

File: app/jobs/cache/browser_channels/responses_for_prefix.rb [hunk 2]
```diff
@@ -163,12 +163,6 @@ def get_site_banner_details(site_banner_lookup)
       details[key.underscore] = value
     end
 
-    if site_banner_lookup.derived_site_banner_info["donationAmounts"].present? && site_banner_lookup.derived_site_banner_info["donationAmounts"] != SiteBanner::DEFAULT_AMOUNTS
```
[the remark this answers is not in the document]
Comment (jlbyrne) [reply app/jobs/cache/browser_channels/responses_for_prefix.rb:166]: We would have, coming from `publishers/protos/channel_responses.rb:19`, but that's one of the deleted lines.  I think github squashed that as a 'large diff' or something.

**Review by tsmartt — APPROVED** [review]

**Pull request merged**